requirements/requirements-base.txt

```diff
@@ -1,4 +1,4 @@
 six>=1.10
 djangorestframework>=3.7.0
-django-rest-swagger>0.3.9,<2.0.0
+django-rest-swagger>=2.2.0
```

Comment:
Does it works good? I think we had problems with 2.0

Reply:
Not everything is displayed in the new swagger, so basically we should scrap the autodoc functionality in the future. But it works (it is not raising any errors).